extension/e2e-tests/sendPayment.test.ts

```diff
@@ -640,208 +640,354 @@ test("Send token payment to C address", async ({ page, extensionId }) => {
   expect(accountBalancesRequestWasMade).toBeTruthy();
 });
 
-test("SendPayment persists amount and asset when navigating to choose address", async ({
+// Classic token to G address -> Normal (regression test)
```

Comment:
looks like there was a bad merge here and overwrote some tests that were here. I'd suggest using `git rebase` instead of merging the release branch. It will give you a cleaner commit history and help you catch issues like this